Makefile

```diff
@@ -36,3 +39,15 @@ release: iso get_gh-release
 	cp $(BUILD_DIR)/minishift-b2d.iso release/
 	$(BUILD_DIR)/gh-release checksums sha256
 	$(BUILD_DIR)/gh-release create minishift/minishift-b2d-iso $(VERSION) master v$(VERSION)
+
+$(BIN_DIR)/minishift:
+	@echo "Downloading latest minishift binary..."
+	@mkdir -p $(BIN_DIR)
+	@cd $(BIN_DIR) && \
+	curl -LO --progress-bar $(MINISHIFT_LATEST_URL) && \
+	tar xzf $(ARCHIVE_FILE)
```

Comment:
Just for the record, this might work for OS X and Linux, but not Windows. Remember there we have a zip.